pkg/yurtmanager/controller/yurtappdaemon/yurtappdaemon_controller.go

```diff
@@ -200,7 +200,7 @@ func (r *ReconcileYurtAppDaemon) Reconcile(_ context.Context, request reconcile.
 		return reconcile.Result{}, err
 	}
 
-	return r.updateStatus(instance, newStatus, oldStatus, currentRevision, collisionCount, templateType, currentNPToWorkload)
+	return r.updateStatus(instance, newStatus, oldStatus, expectedRevision, collisionCount, templateType, currentNPToWorkload)
```

Comment:
I think your changes are correct, and I found the same problem in `yurtappset`, can you help to change it together? cc @luc99hen 
https://github.com/openyurtio/openyurt/blob/d7f64fc295d7a7378d800c37ae83caf5d0570c67/pkg/yurtmanager/controller/yurtappset/yurtappset_controller.go#L209

Reply:
Yes, thanks for reminding!
I will fix another problem.